code/modules/paperwork/paperbin.dm

```diff
@@ -142,7 +142,7 @@
 
 /obj/item/paper_bin/bundlenatural
 	name = "natural paper bundle"
-	desc = "A bundle of paper created using traditional methods."
+	desc = "A bundle of paper created using traditional methods. <b>You can cut the cord on this with a sharp implement, freeing all 30 sheets at once.</b>"
```

Comment:
This description should be added by `examine()` proc.

Reply:
Think I did it correctly